Assets/MRTK/Examples/Experimental/ColorPicker/Scripts/ColorPicker.cs

```diff
@@ -0,0 +1,352 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using TMPro;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.ColorPicker
+{
+    /// <summary>
+    /// Example script to demonstrate adding sliders to control material values at runtime.
+    /// </summary>
+    public class ColorPicker : MonoBehaviour, IMixedRealityTouchHandler
+    {
+        private MeshRenderer targetObjectMesh = null;
+        public MeshRenderer TargetObjectMesh
+        {
+            get => targetObjectMesh;
+            set => targetObjectMesh = value;
+        }
+        private SpriteRenderer targetObjectSprite = null;
+        public SpriteRenderer TargetObjectSprite
+        {
+            get => targetObjectSprite;
+            set => targetObjectSprite = value;
+        }
+        [SerializeField]
+        [Tooltip("")]
+        private MeshRenderer[] PickerUIMeshes = null;
+        [SerializeField]
+        [Tooltip("")]
+        private SpriteRenderer[] PickerUISprites = null;
+        [SerializeField]
+        [Tooltip("")]
+        private MeshRenderer GradientMesh = null;
+        [SerializeField]
+        [Tooltip("")]
+        private GameObject GradientDragger = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderRed = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderGreen = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderBlue = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderAlpha = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderHue = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderSaturation = null;
+        [SerializeField]
+        [Tooltip("")]
+        private PinchSlider SliderBrightness = null;
+        //
+        [SerializeField]
+        [Tooltip("")]
```

Comment:
Out of curiosity, why do these all have empty tooltips?

Reply:
Those were placeholders because I wanted to get the functionality dialed in before writing the tooltip. Fixed